Civi/Api4/Action/WorkflowMessageExample/Get.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
@totten - we discussed this commit yesterday & specifically the slightly odd name - ie `WorkflowMessageExample::get()` and you suggested that @colemanw would be the only one likely to have another opinion

Looking at this PR is strikes me that an entity of  'SampleData::get()` would potentially be useful - since in this specific example the sampleData is just a contact and not tied to the workflow message

However, I suspect you feel that we DO need sample data specific to the message and that perhaps it could call a more generic `sampleData ` in future.

I would probably choose `WorkflowMessageSampleData` as the entity after thinking about it - but I think @colemanw should have the last work on,

Based on my delving to date I would merge this commit if it were in it's own PR & we had a +1 from @colemanw on the naming

Reply:
OK - let's see if @colemanw pipes up....

Reply:
@totten I went through the process of adding sample data for the 2 entities that I am testing this with - going htrough the process does bring me out on the side of `SampleData::get()`

My other thought was that it would be able to specify simpy the entity and get something that represents a very common set of fields - in the case of contact 'alex' in the case of 'contribution' - a Completed contribution like the data I put up. My recurring example is set up to use lots of fields - but that meant choosing a cancelled status - which seems like it isn't the 'normal' scenario
 
https://github.com/civicrm/civicrm-core/pull/21346

Reply:
+1 to what @eileenmcnaughton said. Maybe it's as simple as renaming the API to `SampleData` now and then adding non non-workflow-msg functionality in the future.